history/titles/c_darkmist.txt

```diff
@@ -1,7 +1,12 @@
 1.1.1={
-	law = agnatic_succession
 	liege=d_woodpaw
 }
-345.1.1={holder=29145} #Ro'Bark[12015]
-366.10.04={holder=29147} #Throbal[12015]
-574.1.1={holder=29160} #Gaih[12015]
\ No newline at end of file
+#First dynasty
+320.9.15={holder=29145} #Ro'Bark[12018]
+366.10.4={holder=29147} #Throbal[12018]
+395.9.23={holder=29148} #Daigrron[12018]
+#Second dynasty
+549.7.23={holder=29160} #Gaih[12015]
```

Comment:
And who holds it "in between"?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/76851072-b36f1100-6859-11ea-9596-844372a10b08.png)

</details>

Reply:
No one, it's something from the good old times when we wanted to add the War of the Three Hammers.

Reply:
Done in a5f6a6cfd8eb467affd4ec75486e04bce502786f.